app/Http/Controllers/Bookings/BookingRequestController.php

```diff
@@ -282,23 +282,34 @@ public function list()
 
     private function reservationValidate(Request $request)
     {
+      
         $request->validate(array(
             'reservations.*' => ['array', 'size:3',
                 function ($attribute, $value, $fail) use ($request) {
+                    if(!$request->has('event'))
+                    {
+                        $alcohol  = false;
+                        $food = false;
+                    }
+                    else {
+                        $alcohol  = $request->input('event.alcohol') ?? false;
+                        $food = ($request->input('event.food.low_risk') || $request->input('event.food.high_risk'));
+                    }
                     unset($attribute);
                     $user =  $request->user();
                     $room = Room::query()->findOrFail($request->room_id);
                     $room->minimumReservationTime($value['start_time'], $value['end_time'], $fail);
+                    $room->verifyDatesAreWithinRoomRestrictionsValidation($value['start_time'], $alcohol, $food, $fail, $user);
                     if (!$user->hasPermissionTo('bookings.restrictions.override')) {
-                        $room->verifyDatesAreWithinRoomRestrictionsValidation($value['start_time'], $fail, $user);//
+                        $room->verifyDatesAreWithinRoomRestrictionsValidation($value['start_time'], $alcohol, $food, $fail, $user);
```

Comment:
This should not be twice. Either in or out. and i am pretty sure it goes in.

Reply:
you are completly right fixed.